pkg/agent/route/route_linux.go

```diff
@@ -1245,6 +1258,59 @@ func (c *Client) initWireguard() {
 	}
 }
 
+func (c *Client) initNodeLatency() {
+	gateway := c.nodeConfig.GatewayConfig.Name
+	if c.networkConfig.TrafficEncapMode.IsNetworkPolicyOnly() {
+		gateway = c.networkConfig.TransportIface
+	}
+	icmpZero := int32(0)
```

Comment:
maybe use https://pkg.go.dev/golang.org/x/net/ipv4#ICMPType since I notice that we already import / use this package in `pkg/agent/monitortool`. Same for IPV6.